client/signup/jetpack-connect/index.jsx

```diff
@@ -465,4 +470,9 @@ export default connect(
 		goToPluginInstall,
 		goToPluginActivation
 	}, dispatch )
+);
+
+export default flowRight(
```

Comment:
@sirreal yes. To be more accurate, I'd do this one `export default connect( ... )( localize( Component ) )`. When there's just one HoC other than `connect` it feels simpler, and avoids the import of `flowRight`. Again, I'm fine with using `flowRight` too.